examples/combined_ODNP.py

```diff
@@ -99,19 +111,25 @@
 powers = 1e-3 * 10 ** (dB_settings / 10.0)
 # }}}
 # {{{ these change if we change the way the data is saved
-IR_postproc = "spincore_IR_v3"  # note that you have changed the way the data is saved, and so this should change likewise!!!!
+IR_postproc = "spincore_IR_v3"  # note that you have changed the way the data
+#                                 is saved, and so this should change
+#                                 likewise!!!!
 Ep_postproc = "spincore_ODNP_v5"
 # }}}
 # {{{check total points
 total_points = len(Ep_ph1_cyc) * nPoints
 assert total_points < 2**14, (
-    "For Ep: You are trying to acquire %d points (too many points) -- either change SW or acq time so nPoints x nPhaseSteps is less than 16384\nyou could try reducing the acq_time_ms to %f"
+    "For Ep: You are trying to acquire %d points (too many points) -- either\
+    change SW or acq time so nPoints x nPhaseSteps is less than 16384\nyou\
+    could try reducing the acq_time_ms to %f"
     % total_points,
     config_dict["acq_time_ms"] * 16384 / total_points,
 )
 total_pts = len(IR_ph2_cyc) * len(IR_ph1_cyc) * nPoints
 assert total_pts < 2**14, (
-    "For IR: You are trying to acquire %d points (too many points) -- either change SW or acq time so nPoints x nPhaseSteps is less than 16384\nyou could try reducing the acq_time_ms to %f"
+    "For IR: You are trying to acquire %d points (too many points) -- either\
+    change SW or acq time so nPoints x nPhaseSteps is less than 16384\nyou\
+    could try reducing the acq_time_ms to %f"
```

Comment:
If you try printing this in ipython (always a good idea), you will see that it's a bit weird.  I would recommend:
```
mystring = ("this "
                     "does not have "
                     "weird spaces. "
                     "But, pay attention "
                     "to how I added "
                     "the space at the end "
                     "where needed.")
```